### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vladyslava4**
- [DEMO LINK](https://vladyslava4.github.io/layout_catalog/)
- [TEST REPORT LINK](https://vladyslava4.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,21 +7,377 @@
...
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating rating">
+              <div class="rating__stars stars">
```
Comment (anastasiiavorobiova) [inline src/index.html:133]:
```suggestion
              <div class="rating__stars stars stars--4">
```

File: src/index.html [hunk 2]
```diff
@@ -7,21 +7,377 @@
...
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
```
Comment (anastasiiavorobiova) [inline src/index.html:137]:
```suggestion
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
```
Consider using classes from the previous task

File: src/index.html [hunk 3]
```diff
@@ -7,21 +7,377 @@
...
+                <div class="stars__star"></div>
+              </div>
+              <div class="rating__reviews">Reviews: 5</div>
```
Comment (anastasiiavorobiova) [inline src/index.html:140]:
```suggestion
              <p class="rating__reviews">Reviews: 5</p>
```

File: src/index.html [hunk 4]
```diff
@@ -7,21 +7,377 @@
...
+            data-qa="card"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
```
Comment (anastasiiavorobiova) [inline src/index.html:130]:
```suggestion
            <h2 class="card__title">
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
            </h2>
```
the h1 could be used once per page

File: src/index.html [hunk 5]
```diff
@@ -7,21 +7,377 @@
...
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="rating__reviews">Reviews: 5</div>
+            </div>
+            <div class="card__price price">
+              <div class="price__text">Price:</div>
+              <div class="price__amount">$2,199</div>
+            </div>
```
Comment (anastasiiavorobiova) [inline src/index.html:145]:
```suggestion
            <p class="card__price price">
              <span class="price__text">Price:</span>
              <span class="price__amount">$2,199</span>
            </p>
```
Consider leveraging semantic tags for the text

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,238 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:7]: Consider moving general styles to a separate file

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,238 @@
...
+  border: 1px solid #f3f3f3;
+  display: block;
+  transition: transform 0.3s;
+}
+
+.card:hover {
+  transform: scale(1.2); /* Увеличиваем масштаб */
+}
+
+.card__image {
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:117]: Consider leveraging scss features
```
.card {
    ...

  &__image {
    ...
  }
}
```

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,238 @@
...
+  text-decoration: none;
+
+  height: 40px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  border-radius: 5px;
+  color: #fff;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:169]: Consider using variables for repeating colors

File: src/styles/index.scss [hunk 9]
```diff
@@ -1,3 +1,238 @@
...
+.stars__star {
+  height: 16px;
+  width: 16px;
+
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star--active {
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:207]: The number of active stars should depend on the class with modifier

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Make sure to deploy your solution

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing previous comments and deploying your solution

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Feel free to ask for help in the chat to fix comments and deploy your solution

File: src/index.html (hunk 1 above)
Comment (vladyslava4) [reply src/index.html]: @anastasiiavorobiova  я исправила что не так ?

File: src/index.html [hunk 10]
```diff
@@ -7,21 +7,377 @@
...
```
Comment (anastasiiavorobiova) [inline src/index.html:178]:
```suggestion
            <div class="card__rating rating">
              <div class="rating__stars stars stars--4">
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
              </div>
              <p class="rating__reviews">Reviews: 5</p>
            </div>
            <p class="card__price price">
              <span class="price__text">Price:</span>
              <span class="price__amount">$2,199</span>
            </p>
```
Consider fixing same issues for your blocks

File: src/index.html [hunk 11]
```diff
@@ -7,21 +7,377 @@
...
+          </div>
+          <div class="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
```
Comment (anastasiiavorobiova) [inline src/index.html:195]: Also not fixed

File: src/styles/index.scss (hunk 6 above)
Comment (anastasiiavorobiova) [reply src/styles/index.scss:7]: Not fixed

File: src/styles/index.scss (hunk 7 above)
Comment (anastasiiavorobiova) [reply src/styles/index.scss:118]: Not fixed

File: src/styles/index.scss (hunk 9 above)
Comment (anastasiiavorobiova) [reply src/styles/index.scss:208]: Not fixed

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Screenshot 2024-09-04 at 17 44 07

The demo link leads to the 404 page 
Previous comments are not fixed 
Please, ask for help in the chat or welcome to the Q&A session for help

**Review by etojeDenys — CHANGES_REQUESTED** [review]
pls, don't hesitate to ask for some help with deploy in the chat

**Comment by vladyslava4** [comment]
> пожалуйста, не стесняйтесь обращаться за помощью в развертывании в чате

Знімок екрана 2024-09-04 о 17 52 42
я зверталась мені не відповіли

File: src/index.html [hunk 12]
```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header-content">
```
Comment (volodymyr-soltys97) [inline src/index.html:27]: Redundant
```suggestion
```

File: src/index.html [hunk 13]
```diff
@@ -22,6 +22,359 @@
...
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <div class="catalog__content">
```
Comment (volodymyr-soltys97) [inline src/index.html:115]: Redundant
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, there is no styles on the demo link 
image

**Pull request closed without merging**